projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -3041,9 +3041,6 @@ public static function get_default_label_from_type( $type ) {
 			case 'textarea':
 				$str = __( 'Message', 'jetpack' );
 				break;
-			case 'checkbox':
```

Comment:
Sorry I’m not familiar with the change, but why this removal?

Reply:
In the editor, the checkbox field doesn't have a default label value. 
We need to remove this piece of code to match the same behavior in the backend